index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/style.css">
```

Comment:
@officailesse As highlighted by previous reviewers, your CSS file is not parsing because you have included the forward slash `/` before `style.css` Please remove it to display your design as shown in the below screenshot. 

![afterremovingslash](https://user-images.githubusercontent.com/87413774/164779924-f97b1c2e-e849-4b3d-b5d3-b4c0505afce8.png)